README.md

```diff
@@ -433,6 +433,21 @@ To learn more about model quantization, [read this documentation](examples/quant
 
     </details>
 
+## [`llama-run`](examples/run)
+
+#### A comprehensive example for running `llama.cpp` models. Useful for inferencing. Used with RamaLama [^3].
+
+- <details>
+    <summary>Run a model with a specific prompt</summary>
+
+    ```bash
+    llama-run granite-code
+    >
```

Comment:
```suggestion
```

Reply:
Also not clear where granide-code is downloaded from, what quant is selected, etc.

Reply:
Default is from ollama repo if a file with this name doesn't exist in the current directory and no protocol is specified

Reply:
That's ok, but I would still appreciate some documentation about this, and not assume that the user has knowledge of the way ollama works. For instance, I have no idea what I would need to do to request a different quant.